e2e/test/provisioning/ProvisioningE2ETests.cs

```diff
@@ -313,6 +313,7 @@ public async Task ProvisioningDeviceClient_CustomAllocationPolicy_Mqtt_Symmetric
         }
 
         [LoggedTestMethod]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "<Pending>")]
```

Comment:
Move namespace to usings and fill in Justification.

Reply:
Or is this now covered in the GlobalSuppressions.cs file?

Reply:
fixing.